components/Header/HeaderCTA.tsx

```diff
@@ -68,8 +70,9 @@ const HeaderCTA = () => {
         </div>
         <Button
           as="link"
-          href="/pricing/"
+          href="https://goteleport.com/pricing/"
```

Comment:
Why adding host? Generally absolute paths are best, so they are domain agnostic.

Reply:
I've updated this href to be `https://${HOST}/pricing/` to align with [`structure`](https://github.com/gravitational/docs/blob/8902141643a26f692b8f79507e2155c5e6c8a636/components/Menu/structure.ts#L31) of the other external links in navigation bar.

Reply:
@sandylcruz What is the reason that the structure there though? This means that localhost and goteleport.dev links are incorrect, as an example. If there isn't

Reply:
Updated